base/multi.jl

```diff
@@ -1595,7 +1600,13 @@ function check_master_connect()
         end
 
         if !haskey(map_pid_wrkr, 1)
-            print(STDERR, "Master process (id 1) could not connect within $timeout seconds.\nexiting.\n")
+            print(STDERR, """
+                          Master process (id 1) could not connect within $timeout seconds.
+                          If connecting over SSH, please ensure the host has an unbound,
+                          accessible port above or including port $WORKER_LISTEN_PORT
+                          for the worker process to connect to.
+                          Exiting.
+                          """)
```

Comment:
I understand the motivation here, though I am not sure how much details we should print here. 

- `tunnel=true` would allow for the master to connect to WORKER_LISTEN_PORT. Should we suggest that?
- Should we print the value of `actual_port`?
- This error is printed when the master is unable to connect. So the last line should read "for the master process to connect to". However, while the master can still use ssh port forwarding and connect to actual_port via 22, all the workers connect to each other directly. Should we mention that too?

I guess, the larger question I am asking here is should we just have the error message point to a section in the documentation or should we detail all possible causes and solutions here.